sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py

```diff
@@ -1931,7 +1938,8 @@ async def set_blob_tags(self, tags=None, **kwargs):
         :returns: Blob-updated property dict (Etag and last modified)
         :rtype: Dict[str, Any]
         """
-        options = self._set_blob_tags_options(tags=tags, **kwargs)
+        version_id = kwargs.pop('version_id', None) or self.version_id
+        options = self._set_blob_tags_options(tags=tags, version_id=version_id, **kwargs)
```

Comment:
If you handle this in `_set_blob_tags_options` in sync client. You won't need to worry about it here.

Reply:
I think this comment is still relevant?